app/models/task.rb

```diff
@@ -264,6 +264,34 @@ def do_hold(last_check)
     end
   end
 
+  def name
```

Comment:
I don't think you need this method. We already have a `#type_name_display` method which formats the Task type's name for display that more or less gives the same results.